ios/imagemounter/personalized_image_mounter.go

```diff
@@ -0,0 +1,196 @@
+package imagemounter
+
+import (
+	"fmt"
+	"github.com/Masterminds/semver"
+	"github.com/danielpaulus/go-ios/ios"
+	log "github.com/sirupsen/logrus"
+	"io"
+	"os"
+	"path"
+)
+
+type PersonalizedDeveloperDiskImageMounter struct {
+	deviceConn ios.DeviceConnectionInterface
+	plistRw    ios.PlistCodecReadWriter
+	version    *semver.Version
+	tss        tssClient
+	ecid       uint64
+}
+
+func NewPersonalizedDeveloperDiskImageMounter(entry ios.DeviceEntry, version *semver.Version) (PersonalizedDeveloperDiskImageMounter, error) {
+	values, err := ios.GetValuesPlist(entry)
+	if err != nil {
+		return PersonalizedDeveloperDiskImageMounter{}, nil
+	}
+	var ecid uint64
+	if e, ok := values["UniqueChipID"].(uint64); ok {
+		ecid = e
+	} else {
+		return PersonalizedDeveloperDiskImageMounter{}, fmt.Errorf("could not get ECID from device")
+	}
+	deviceConn, err := ios.ConnectToService(entry, serviceName)
+	if err != nil {
+		return PersonalizedDeveloperDiskImageMounter{}, err
+	}
+	return PersonalizedDeveloperDiskImageMounter{
+		deviceConn: deviceConn,
+		plistRw:    ios.NewPlistCodecReadWriter(deviceConn.Reader(), deviceConn.Writer()),
+		version:    version,
+		tss:        newTssClient(),
+		ecid:       ecid,
+	}, nil
+}
+
+func (p PersonalizedDeveloperDiskImageMounter) Close() error {
+	return p.deviceConn.Close()
+}
+
+func (p PersonalizedDeveloperDiskImageMounter) ListImages() ([][]byte, error) {
+	return listImages(p.plistRw, "Personalized", p.version)
+}
+
+func (p PersonalizedDeveloperDiskImageMounter) MountImage(imagePath string) error {
+	manifest, err := loadBuildManifest(path.Join(imagePath, "BuildManifest.plist"))
+	if err != nil {
+		return fmt.Errorf("failed to load build manifest. %w", err)
+	}
+
+	identifiers, err := p.queryIdentifiers()
+	if err != nil {
+		return fmt.Errorf("failed to query personalization identifiers. %w", err)
```

Comment:
same here and all errors below, I won'T add comments for each one. Please check it yourself.